### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by smira**
| Package | Update | Change |
|---|---|---|
| [containerd/containerd](https://togithub.com/containerd/containerd) | patch | `v1.7.13` -> `v1.7.14` |
| [containernetworking/plugins](https://togithub.com/containernetworking/plugins) | patch | `v1.4.0` -> `v1.4.1` |
| git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git | minor | `20240220` -> `20240312` |
| git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git | patch | `6.6.20` -> `6.6.21` |
| git://git.kernel.org/pub/scm/utils/cryptsetup/cryptsetup.git | patch | `2.7.0` -> `2.7.1` |
| [siderolabs/bldr](https://togithub.com/siderolabs/bldr) | minor | `v0.2.3` -> `v0.3.0` |
| [tukaani-project/xz](https://togithub.com/tukaani-project/xz) | patch | `v5.6.0` -> `v5.6.1` |
| git://git.kernel.org/pub/scm/utils/kernel/kmod/kmod.git | major | `31` -> `32` |

### CODE REVIEW

File: Pkgfile [hunk 1]
```diff
@@ -6,20 +6,20 @@ vars:
   TOOLS_IMAGE: ghcr.io/siderolabs/tools:v1.7.0-alpha.0-10-g14bf457
```
Comment (frezbo) [inline Pkgfile:6]: bldr version update here also

**Review by frezbo — APPROVED** [review]

File: Pkgfile (hunk 1 above)
Comment (smira) [reply Pkgfile:6]: done, thanks

**Pull request merged**